fastlane/lib/fastlane/actions/ensure_no_debug_code.rb

```diff
@@ -22,6 +22,12 @@ def self.run(params)
 
         command << " --exclude #{params[:exclude]}" if params[:exclude]
 
+        if params[:exclude_dirs]
```

Comment:
I'm generally more of a fan of the if when it comes to debugging, also removes a no op allocation vs creating an array.